distributed/client.py

```diff
@@ -130,6 +130,13 @@
 TOPIC_PREFIX_FORWARDED_LOG_RECORD = "forwarded-log-record"
 
 
+class SourceCode(NamedTuple):
```

Comment:
nit: Maybe `Frame` might be a better name? If so, I'd also suggest renaming `lineno_frame` to `firstlineno`.

❤️ for the typed `NamedTuple`!